python/ray/llm/_internal/serve/deployments/prefill_decode_disagg/prefill_decode_disagg.py

```diff
@@ -37,21 +35,18 @@ class PDServingArgs(BaseModel):
     prefill_config: Union[str, LLMConfig]
     decode_config: Union[str, LLMConfig]
 
+    @field_validator("prefill_config", "decode_config")
+    @classmethod
+    def validate_configs(cls, value):
+        return LLMConfig.parse_from([value])[0]
+
     def parse_args(self) -> "PDServingArgs":
```

Comment:
we don't need parse_args anymore. Simply we can do `PDServingArgs(prefill_config=x, decode_config=y)`

or `PDServingArgs.model_validate(pd_serving_args)`